CHANGE-NOTES.html

```diff
@@ -1,19 +1,20 @@
 Features:
 <ul>
-  <li>Added support for pulling current branch</li>
-  <li>Added support for sliding in a new branch (or reattaching a existing one) without the need to edit machete file manually</li>
-  <li>Open Machete Tab action was added under VCS/Git submenu with key shortcut Ctrl + Alt + Shift + M</li>
-  <li>Current branch is marked even if there is an ongoing operation in repo (rebasing, bisecting, etc.) and name of this operation is displayed</li>
-  <li>Added a notification when skipping branches declared in machete file but nonexistent in the repository</li>
-  <li>Reset action is preceded by information dialog</li>
-  <li>Reset is prohibited when uncommitted changes are present in repository</li>
+  <li>Added discover functionality</li>
+  <li>Added fork point override functionality</li>
+  <li>Branch reset is possible without checking it out</li>
+  <li>Toolbar menu revamp to provide most suitable actions</li>
```

Comment:
Use full sentences (with a subject&verb):
```suggestion
  <li>Toolbar menu was revamped to suggest most suitable actions for the current branch</li>
```